src/components/ListField/ListField.test.jsx

```diff
@@ -176,185 +167,116 @@ describe('ListField - Collaborators', () => {
-  });
-
-  it.skip('correctly handles removing members of the item', () => {
-    // TODO: convert to RTL
-    const component = render(<ListField {...collaboratorDefaultProps} />);
-    let collaborators = component.find('.mock-collaborator');
-    expect(collaborators).toHaveLength(collaboratorInput.value.length);
-
-    const firstCollaborator = component.state().currentList[0];
-    // Petend we deleted the first collaborator
-    const firstUuid = collaboratorInput.value[0].uuid;
-    component.instance().handleRemove(firstUuid);
-
-    // Verify that the onChange method has been called
-    expect(collaboratorInput.onChange).toBeCalled();
-
-    // Verify that the first collaborator has been removed
-    component.update();
-    collaborators = component.find('.mock-collaborator');
-    expect(collaborators).toHaveLength(collaboratorInput.value.length - 1);
-
-    const newFirstCollaborator = component.state().currentList[0];
-    expect(firstCollaborator).not.toEqual(newFirstCollaborator);
-  });
-
-  it.skip('correctly handles reordering members', () => {
-    // TODO: convert to RTL
-    const component = render(<ListField {...collaboratorDefaultProps} />);
-    // Find the first item.
-    const firstItem = component.state().currentList[0].uuid;
-
-    const result = {
-      source: {
-        index: 0,
-      },
-      destination: {
-        index: 2,
-      },
-    };
-    // Pretend we dragged the first item to the end.
-    component.instance().onDragEnd(result);
-
-    // Verify that the onChange method has been called
-    expect(collaboratorInput.onChange).toBeCalled();
-
-    // Verify that it is on the end.
-    expect(firstItem).toEqual(component.state().currentList[2].uuid);
+  it('adds the referred item to state when one is given', async () => {
+    render(<MemoryRouter><ListField {...collaboratorReferredProps} /></MemoryRouter>);
+    const listItems = await screen.findAllByTestId('draggable-list-item');
+    const lastItem = listItems[listItems.length - 1];
+    expect(lastItem).toHaveTextContent('I am a school');
   });
 
-  it.skip('does not re-order when dragged outside of the list', () => {
-    // TODO: convert to RTL
-    const component = render(<ListField {...collaboratorDefaultProps} />);
-    // Find the first item.
-    const firstItem = component.state().currentList[0].uuid;
-
-    const result = {
-      source: {
-        index: 0,
-      },
-    };
-    // Pretend we dragged the first item outside the list.
-    component.instance().onDragEnd(result);
-
-    // Verify that the onChange method has NOT been called
-    expect(collaboratorInput.onChange).not.toBeCalled();
-    expect(firstItem).toEqual(component.state().currentList[0].uuid);
+  it('gets no suggestions for short autocomplete', async () => {
+    render(<MemoryRouter><ListField {...collaboratorDefaultProps} /></MemoryRouter>);
+    const input = await screen.getByRole('textbox');
```

Comment:
nit: I noticed we're using `await` with `screen.getBy*` in some places, but not in others. Since getBy* is a synchronous method and doesn't require `await`, could we remove it where it's not needed and make the usage consistent throughout the file?

Reply:
Ah, I was facing issues with these two tests, so I tried to fix them using await. In the end, it turned out to be a different issue, which I resolved. But I forgot to remove these calls. Thanks for pointing it out.
Lemme remove them from these tests real quick